basic/exercise11/scriptSpec.js

```diff
@@ -1,15 +1,15 @@
 /// <reference path="../../typings/globals/jasmine/index.d.ts" />
 
-describe("Checking if  degrees Celsius ", function(){
-    it("is degrees Fahrenheit", function(){
-        var temperatureInDegreesFahrenheit =  convertTemperaturesFromCelsiusToFahrenheit(degreesCelsius);
-        expect(temperatureInDegreesFahrenheit).toBeTruthy();
+describe("10 degrees Celsius" , function(){
```

Comment:
10 should not be in describe section but in it section - describe section should aggregate many tests and each test instantiated by it section should have different parameter